apps/deploy-web/src/components/authorizations/GranteeRow.tsx

```diff
@@ -15,6 +15,7 @@ type Props = {
 
 export const GranteeRow: React.FunctionComponent<Props> = ({ grant }) => {
   const limit = grant?.authorization?.spend_limit;
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   const denomData = limit ? useDenomData(limit.denom) : null;
```

Comment:
this is a buggy place, need to fix, because hooks cannot be conditionally called